packages/components/psammead-media-player/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-media-player",
-  "version": "1.0.1-alpha.1",
+  "version": "1.0.1",
```

Bumping an alpha version to stable.
```suggestion
  "version": "1.0.1-alpha.2",
```
I have a PR to fix the alpha versioning https://github.com/bbc/psammead/pull/1949